package.json

```diff
@@ -5,22 +5,7 @@
     "node": ">=10"
   },
   "dependencies": {
-    "@primer/octicons": "9.6.0",
-    "clipboard": "2.0.6",
-    "dropzone": "5.7.0",
-    "fomantic-ui": "2.8.4",
-    "highlight.js": "9.18.1",
-    "jquery": "3.4.1",
-    "jquery-datetimepicker": "2.5.21",
-    "jquery-migrate": "3.1.0",
-    "jquery.are-you-sure": "1.9.0",
     "sortablejs": "1.10.2",
-    "swagger-ui": "3.25.0",
-    "vue": "2.6.11",
-    "vue-bar-graph": "1.2.0",
-    "vue-calendar-heatmap": "0.8.4"
-  },
-  "devDependencies": {
```

Comment:
This seems wrong. We should have pretty much all dependencies in `dependencies`, except linters and `updates`.